bindings/wallet-core/src/c/vote.rs

```diff
@@ -6,7 +6,7 @@ use std::convert::{TryFrom, TryInto};
 use std::ffi::CStr;
 pub use wallet::Settings;
 
-const ENCRYPTION_VOTE_KEY_HRP: &str = "p256k1_votepk";
+const ELECTION_PUBLIC_KEY_HRP: &str = "votepk";
```

Comment:
I had an idea that key algorithm discrimination in the BECH is useful to avoid confusing errors and provide better troubleshooting clue. It's probably not important for the relatively short-lived election keys, and the worst that can happen is that an invalid key would be faulted just for being invalid, not specifically because it was generated with a different implementation.

Reply:
Agree that it would be better to have it in the name, but I don't see an easy way to do this. 
OTOH, this is never actually seen/used directly by users and it's not something you expect to change often (and anyway will never change in the context of a single fund). 
There is actually some potential for interpreting a key on a different curve depending on the format used (e.g. if only the x coordinate is encoded and same endianess), but this is not the case for current backends